.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: build
+
+on:
+  push:
+    branches:
+      - release/**
+
+jobs:
+  dist:
+    name: distribution packages
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - run: |
+          python -m pip install --upgrade pip
```

Comment:
This may not be a great idea. Why always use the latest `pip`, instead of pinning a version?

Reply:
I copied this from the GHA Python template. I have no strong feelings about it, just one less package I have to update periodically.

Reply:
I'm fairly sure this will break your builds unexpectedly some time in the future but it should be rare :)